packages/clerk-js/src/utils/url.ts

```diff
@@ -350,3 +350,22 @@ export const isAllowedRedirectOrigin = (_url: string, allowedRedirectOrigins: Ar
   }
   return isAllowed;
 };
+
+export function createAllowedRedirectOrigins(
+  allowedRedirectOrigins: Array<string | RegExp> | undefined,
+  frontendApi: string,
+): (string | RegExp)[] | undefined {
+  if (!allowedRedirectOrigins || allowedRedirectOrigins.length === 0) {
+    const origins = [];
+    if (typeof window !== 'undefined' && !!window.location) {
```

Comment:
❓ could we use the `inBrowser()` helper instead? @anagstef wdyt?

Reply:
Oh I had that but change it as another utility function in the same file had it like this so I stick with that for consistency, but I can change the `inBrowser` for every function here.